src/app/state/auth/auth.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+export interface AuthResponse {
```

Comment:
Não defina interfaces no próprio componente, exceto se for usado apenas por ele; e nesses casos, `export` torna-se desnecessário. Prefira criar interfaces no diretório destinado para isso.